universum/modules/output/github_output.py

```diff
@@ -0,0 +1,61 @@
+from .base_output import BaseOutput
+
+__all__ = [
+    "GithubOutput"
+]
+
+
+class GithubOutput(BaseOutput):
+    """
+    GitHub doesn't support nested grouping
+    See: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines
```

Comment:
I think the more correct link would be https://github.com/actions/runner/issues/802

Reply:
done